src/main/kotlin/io/kotest/datagen/wine/wine.kt

```diff
@@ -0,0 +1,60 @@
+package io.kotest.datagen.wine
+
+import io.kotest.datagen.Name
+import io.kotest.datagen.NameProducer
+import io.kotest.datagen.Producer
+import kotlin.random.Random
+
+object VineyardProducer : Producer<Vineyard> {
+   private val vineyards = javaClass.getResourceAsStream("/wine/vineyards.txt").bufferedReader().readLines()
```

Comment:
This `javaClass.getResourceAsStream ... ` is repeated quite a lot, perhaps extracting it is a good idea?

Reply:
I can't move it to a top level function because I need access to the classloader.

Reply:
I guess I can do this:
fun loadResource(resource: String) = object {}.javaClass.getResourceAsStream(resource)

Reply:
Ok done in master and will merge into this after.